### PR TITLE
Setup Prettier

### PR DESCRIPTION
**Pull request by zg3d**
Fixes issue #157 

-added .pretterrc.json
-added prettier script in package.json
-the script was ran on the repo
- added formatting command in contributing document

Please let me know if I need to make changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "start": "ember serve",
     "test": "ember test",
     "test:all": "ember try:each",
-    "test:node": "mocha node-tests/**/*-test.js"
+    "test:node": "mocha node-tests/**/*-test.js",
+    "format": "prettier --write \"./**/*.{md,json,js,yml}\""
```
Comment (jelhan) [inline package.json:23]: I would prefer to setup Prettier as ESLint plugin as described in the RFC: https://emberjs.github.io/rfcs/0628-prettier.html#detailed-design My main motivation is that we don't need to change much when this RFC is implemented in Ember CLI.

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "semi": true,
+  "tabWidth": 2,
+  "bracketSpacing": false
+}
```
Comment (jelhan) [inline .prettierrc.json:8]: Any reasons for this specific configuration? I would prefer to use the configuration as defined by Prettier RFC: https://emberjs.github.io/rfcs/0628-prettier.html#prettier Mostly to use the same as other Ember addons.

**Review by jelhan — CHANGES_REQUESTED** [review]
Thanks a lot for picking this one up! :clap:

Would it okay for you to align it more with Prettier RFC? I think this would be more future proof and cause less noise when Prettier is added to default blueprints.

**Comment by zg3d** [comment]
I'll implement all the changes

**Review by jelhan — APPROVED** [review]
Thanks a lot!

**Pull request merged**